docs/tutorials/introduction.ipynb

```diff
@@ -161,9 +161,10 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "sc.table(d['alice']) # includes coordinates\n",
-    "sc.table(d['alice'].data) # the variable holding the data, i.e., the dimension labels, units, values, and optional variances\n",
-    "sc.table(d['alice'].values) # just the array of values, shorthand for d['alice'].data.values"
+    "from IPython.display import display\n",
```

Comment:
Why did you add calls to `display`

Reply:
Because I changed the function to return the `HTML()` object, instead of calling `display(HTML())` inside the function and returning None. This was more consistent with the html repr, but I can change it back if we prefer the old behaviour.

Reply:
I think the way it works no is better :+1: